### PR TITLE
Added deb/rpm specs

### PR DESCRIPTION
**Pull request by moredure**

### CODE REVIEW

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -55,18 +55,15 @@ jobs:
...
     # ℹ️ Command-line programs to run using the OS shell.
     # 📚 See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun
 
     #   If the Autobuild fails above, remove it and uncomment the following three lines.
     #   modify them (or add more) to build your code if your project, please refer to the EXAMPLE below for guidance.
 
-    # - run: |
-    #   echo "Run, Build Application using script"
-    #   ./location_of_script_within_repo/buildscript.sh
+    - run: |
```
Comment (telepenin) [inline .github/workflows/codeql.yml:66]: Do not clear why Makefile is deleted, could you add comment/name to clarifying?

File: Makefile [hunk 2]
```diff
@@ -1,38 +1,16 @@
...
+install: SBIN_DIR ?= /usr/sbin
+install:
+	mkdir -p "$(DESTDIR)/var/lib/imunify-nsqd"
+	mkdir -p "$(DESTDIR)${SBIN_DIR}"
+	install -m 0755 ${REPO_ROOT}/bin/imunify-nsqd $(DESTDIR)${SBIN_DIR}/imunify-nsqd
 
-$(BLDDIR)/%:
-	@mkdir -p $(dir $@)
-	go build ${BLDFLAGS} -o $@ ./apps/$*
+install-go: GOVERSION ?= go1.19.1
```
Comment (telepenin) [inline Makefile]: Please bump to 1.19.4

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -24,9 +24,6 @@ jobs:
         with:
           go-version: ${{matrix.go}}
 
-      - name: build
```
Comment (telepenin) [inline .github/workflows/test.yml:27]: Now we do not build pkg in workflow. Why?

Comment (moredure) [reply .github/workflows/test.yml:27]: It hadn't built before as well, make had mostly empty all target.

File: .github/workflows/codeql.yml (hunk 1 above)
Comment (moredure) [reply .github/workflows/codeql.yml:66]: Makefile is deleted in order to skip triggering make build by codeanalyze which then fails with downloaded ./go sources and files

Comment (telepenin) [reply .github/workflows/codeql.yml:66]: Thats the reason why we removed, i think to be clear better explicitly say about it and add name for that step.

Comment (moredure) [reply .github/workflows/codeql.yml:66]: Truth be told we may mv it to Makefile.backup and after codeql step return it back, but currently it not make much sense

**Review by telepenin — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (moredure) [reply Makefile]: Done

**Pull request merged**